src/app/server/Server.cpp

```diff
@@ -90,15 +90,29 @@ class ServerStorageDelegate : public PersistentStorageDelegate
 
     CHIP_ERROR SyncGetKeyValue(const char * key, void * buffer, uint16_t & size) override
     {
+        ChipLogProgress(AppServer, "Retrieved value from server storage.");
         return PersistedStorage::KeyValueStoreMgr().Get(key, buffer, size);
     }
 
     CHIP_ERROR SyncSetKeyValue(const char * key, const void * value, uint16_t size) override
     {
-        return PersistedStorage::KeyValueStoreMgr().Put(key, value, size);
+        ChipLogProgress(AppServer, "Stored value in server storage");
+        CHIP_ERROR err = PersistedStorage::KeyValueStoreMgr().Put(key, value, size);
```

Comment:
are these changes needed?

I think `return ... ` may be cleaner to read than assign + return pair (less vertical space)

Reply:
this was to get some logging here. Makes it easier to debug.

Reply:
err is not logged. you can keep

```cpp
return PersistedStorage::KeyValueStoreMgr().Put(key, value, size);
```

and keep the log above.

Reply:
fixed